edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -1093,7 +1111,7 @@ mod tests {
                 "management_uri": "unix:///var/run/iotedge/mgmt.sock",
                 "workload_uri": "unix:///var/run/iotedge/workload.sock"
             },
-            "homedir": "/var/lib/iotedge",
```

Comment:
What's the reason or this change? Would it create any impact to existing features?

Reply:
Also, what if tempdir setting is empty?

Reply:
The reason for this change is for the need to store notary trust collection in ~/home_dir/notary. So need the information of home directory to build notary directory on top of it. It won't affect the existing change. Before it was a fixed directory "/var/lib/iotedge" and now it is auto-generated for each test. Temp_Dir cannot be empty as we are using new() function to get a temp_dir